### DISCUSSION TITLE
Is joomla ready for new browsers without the "user agent", and with the 100 in version number?

### DISCUSSION THREAD

**Discussion by simbus82** [joomla/joomla-cms#37244] [General]
In next months many changes will occours in the browsers world.
Lot's of browser are ready to reach the V100, breaking some rules and some old regex in many scripts that try to read the browsers versions.
https://www.techspot.com/news/93443-version-100-chrome-edge-firefox-may-break-websites.html

Then the end of "user agents" are near, so lot's of scripts and regex based on user agents, they may no longer work
https://www.xda-developers.com/chrome-100-beta/
https://blog.chromium.org/2022/03/chrome-100-beta-reduced-user-agent.html

Is Joomla ready to work with new browsers?
Or does the core contain many references to specific user agents and browser versions?

- A list of ua detection broken scritps to not use https://www.otsukare.info/2022/01/14/broken-ua-detection
- List of Mozilla bugs for v100 https://github.com/webcompat/web-bugs/labels/version100
- Force v100 in Chromium to see bugs https://bugs.chromium.org/p/chromium/issues/detail?id=1273958

**Comment by brianteeman** [comment]
yes
